wom/src/main/scala/wom/transforms/WomCallNodeMaker.scala

```diff
@@ -0,0 +1,13 @@
+package wom.transforms
+
+import common.validation.ErrorOr.ErrorOr
+import wom.graph.CallNode.CallNodeAndNewNodes
+import wom.graph.GraphNodePort
+import simulacrum._
+import scala.language.implicitConversions
+
+@typeclass
+trait WomCallNodeMaker[A] {
+  @op("toWomCallNode")
```

Comment:
These `@op` annotations are used to alias these methods (looks like you are using them as a whitelist?)

I.e. if you omit this you will get the exact same output.

Reply:
ok, I thought that was required to choose the functions you wanted to expose (as opposed to other helper functions I guess), probably just an artifact of copying an example which used them to alias the name.